### DISCUSSION TITLE
A release compatible with trio-0.22?

### DISCUSSION THREAD

**Discussion by mgorny** [agronholm/anyio#527] [General]
The 3.6.2 release is incompatible with trio-0.22. Is there a plan to make a new release that would unblock trio upgrade? I'm asking because we'd like to remove trio-0.21 from Gentoo.

**Comment by agronholm** [comment]
Believe me, I want to make a new release (v4.0), but there are blocker issues for that. I am working on [another project](https://github.com/agronholm/typeguard) now, but I will shift back to AnyIO as soon as I can get the major release out in that other project.

**Comment by davidbrochart** [comment]
BTW thanks a lot for working on so many awesome projects @agronholm.

**Comment by agronholm** [comment]
v4.0.0 final is out now.
